.travis.yml

```diff
@@ -2,9 +2,13 @@ language: ruby
 rvm:
   - 2.3.3
 before_install:
-  - cd site 
+  - cd site
   - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo $super_secret_password | gpg --passphrase-fd 0 service-acc.json.gpg; fi'
-script: 
+script:
+  - nvm install 7
+  - npm install
+  - bundle exec jekyll build
```

Comment:
@chrisdwheatley yeah I see why you have to make sure the service worker file gets moved to the output folder but work locally as well. But I think we can still have one build call here for simplicity.

```
script:
  - bundle exec jekyll build
  - npm run precache
```

For the precache npm command, we can make sure it deploys to _site specifically (take a look at my comment in `package.json`.)